### PR TITLE
Javers uses deprecated type TEXT for mssql for Snapshot_State and Snapshot_Changed fields

### PR DESCRIPTION
**Pull request by wcchristian**
- Adding alter statement for mysql test text -> varchar(max) conversion.

- Setting gradle to pull new version of polyjdbc

- Adding mssql driver for testing  (unsure if this should be committed but I couldn't test without it, intellij was saying that I needed a driver.)

Note: I ran the tests with the build and the mssql integration tests and everything came up green 😄

### CODE REVIEW

**Comment by wcchristian** [comment]
Appears that all PRs are currently failing Travis build with the following message
```
The command "openssl aes-256-cbc -K $encrypted_750aaef1260c_key -iv $encrypted_750aaef1260c_iv -in travis/secring.gpg.enc -out travis/secring.gpg -d" failed and exited with 1 during .
```

File: javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/JaversSchemaManager.java [hunk 1]
```diff
@@ -73,6 +77,14 @@ private void alterCommitIdColumnIfNeeded() {
         }
     }
 
+    /**
+     * This method is needed for upgrading TEXT columns to VARCHAR(MAX) since TEXT is deprecated.
+     */
+    private void alterMssqlTextColumns() {
```
Comment (bartoszwalacik) [inline javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/JaversSchemaManager.java]: this method should be called only once, if col type hasn't been altered yet, see `alterCommitIdColumnIfNeeded()`

Comment (bartoszwalacik) [inline javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/JaversSchemaManager.java]: one more question, did you tested also this schema migration script? I mean did you run new version of javers with old schema?

**Comment by bartoszwalacik** [comment]
you are right about Travis, we need to fix it

**Comment by bartoszwalacik** [comment]
I have pushed a commit to master which should fix the travis issue, if you merge with master, your branch should be green

File: javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/JaversSchemaManager.java (hunk 1 above)
Comment (wcchristian) [inline javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/JaversSchemaManager.java]: Yep, I tested upgrading an old schema by committing to javers, I also added in a check for the type of the column before running the upgrade.

**Comment by wcchristian** [comment]
Ready for another review!

**Comment by bartoszwalacik** [comment]
ok, @wcchristian give me a while, now I'm on devoxx. I will back to you soon

**Comment by wcchristian** [comment]
No problem!

**Comment by wcchristian** [comment]
Awesome, thanks! Excited to make a contribution!

**Comment by bartoszwalacik** [comment]
@wcchristian thanks, v 3.3.1 with your contribution is released https://javers.org/release-notes/

**Comment by wcchristian** [comment]
Awesome! Thanks!

**Pull request merged**